test/tools/libtesteth/ImportTest.cpp

```diff
@@ -205,6 +205,28 @@ set<eth::Network> ImportTest::getAllNetworksFromExpectSections(
     return networkSet;
 }
 
+void ImportTest::parseJsonStrValueIntoSet(json_spirit::mValue const& _json, set<string>& _out)
```

Comment:
Why is this a member of the class and the function below is not?
Make them both free functions in unnamed namespace. Make them return set/vector instead of output params.

Reply:
unnamed no. it is used outside of this file

Reply:
the other one is not used outside